pages/docs/learn/inngest-steps.mdx

```diff
@@ -0,0 +1,164 @@
+import IconConcurrency from 'src/shared/Icons/FlowControl/Concurrency';
+
+export const description = 'Learn about Inngest steps and their methods.';
+
+# Inngest Steps
+
+Steps are fundamental building blocks in Inngest functions. Each step represents individual task (or other unit of work) within a function that can be executed independently.
+
+Steps are crucial because they allow functions to run specific tasks in a controlled and sequential (or parallel) manner. You can build complex workflows by chaining together simple, discrete operations.
+
+On this page, you will learn about the benefits of using steps, and get an overview of the available step methods.
+
+## Benefits of Using Steps
+
+- **Improved reliability**: structured steps enable precise control and handling of each task within a function.
+- **Error handling**: capturing and managing errors at the step level means better error recovery.
+- **Retry mechanism**: failing steps can be retried and recovered independently, without re-executing other successful steps.
+- **Independent testing**: each step can be tested and debugged independently from others.
+- **Improved code readability**: modular approach makes code easier to navigate and refactor.
+
+If you'd like to learn more about how Inngest steps are executed, check the ["How Inngest functions are executed"](/docs/learn/how-functions-are-executed) page.
+
+## Anatomy of an Inngest Step
+
+The first argument of every Inngest step method is an `id`. Each step is treated as a separate job which can be individually retried, debugged, or recovered. Inngest uses the ID to memoize step state across function versions.
```

Comment:
**suggestion**: "job" might feel weird as we might reference the entire function as a "background job" elsewhere.
```suggestion
The first argument of every Inngest step method is an `id`. Each step is treated as a discrete task which can be individually retried, debugged, or recovered. Inngest uses the ID to memoize step state across function versions.
```